server/services/odata.service.js

```diff
@@ -234,11 +234,49 @@ module.exports = class ODataService {
     }
   }
 
-  static async updateRecordAttachments (id, supportingInformation) {
+  static async updatePhotos (isSection2, entity, photoRecords) {
     const token = await ActiveDirectoryAuthService.getToken()
 
+    const id = isSection2
+      ? entity[DataVerseFieldName.SECTION_2_CASE_ID]
+      : entity[DataVerseFieldName.SECTION_10_CASE_ID]
+
     const apiEndpoint = `${config.dataverseResource}/${config.dataverseApiEndpoint}`
+    const patchCommands = []
+    for (let i = 0; i < photoRecords.files.length; i++) {
+      const fieldName = `cre2c_photo${i + 1}`
+      const url = `${apiEndpoint}/${isSection2 ? SECTION_2_ENDPOINT : SECTION_10_ENDPOINT}(${id})/${fieldName}`
+
+      const headers = {
+        'OData-Version': ODATA_VERSION_NUMBER,
+        'OData-MaxVersion': ODATA_VERSION_NUMBER,
+        [AUTHORIZATION]: `Bearer ${token}`,
+        [PREFER]: PREFER_REPRESENTATION,
+        [CONTENT_TYPE]: ContentTypes.APPLICATION_OCTET_STREAM
+      }
+
+      const body = Buffer.from(photoRecords.fileData[i], 'base64')
+
+      console.log(`Patching URL: [${url}]`)
```

Comment:
There shouldn't be `console.log` commands in the code - this should be removed

(I think this line had been left in erroneously from earlier development)